docs/articles/documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+title: Connect to the TestCafe Server over HTTPS
+permalink: /documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.html
+---
+# Connect to TestCafe Server over HTTPS
+
+TestCafe is a proxy-based testing tool. Browser requests are sent via the TestCafe proxy server to the tested website. All requests between the browser and the TestCafe server are sent over the HTTP protocol.
+
+![Connection Protocols](../../../images/proxy-connection-protocols.svg)
+
+Some browser features (like
+[Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API),
+[Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API),
+[ApplePaySession](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession), or
+[SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto))
+require secure origin. This means that the website should be served over the HTTPS protocol. If you run tests against pages with this kind of browser API in a regular way, these tests fail with JavaScript errors.
+
+TestCafe can serve the proxied tested page over the HTTPS protocol. When this option is enabled, the client browser connects to the TestCafe proxy server over HTTPS. This allows you to test webpages that use browser features that require secure origin.
```

Comment:
serve?
"This allows you to test **web pages** that use browser features that require secure origin." that, that?

Reply:
1. `serve?`

    Yes, the server **serves** web pages. You can find this verb even on Wikipedia.
https://en.wikipedia.org/wiki/Static_web_page
https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_web_server#Static_vs._dynamic_content

2. `that... that?`

    Yep, I didn't like it either.

**Version 2**

TestCafe can serve the proxied tested page over the HTTPS protocol. When this option is enabled, the client browser connects to the TestCafe proxy server over HTTPS. This allows you to test web pages with browser features that require secure origin.

Reply:
TestCafe can serve the proxied tested page over the HTTPS protocol. When this option is enabled, the client browser uses HTTPS to connect to the TestCafe proxy server. This allows you to test web pages with browser features that require a secure origin.